### PR TITLE
Implement HttpResponse support (see issue #4)

### PR DESCRIPTION
**Pull request by 1st1**
See issue #4.

### CODE REVIEW

**Comment by asavaritayal** [comment]
Since we're beyond the basic prototype now, let's have @christopheranderson and @pragnagopa review the checkins. Would be good for us to get more familiar with the code and design patterns.

**Comment by 1st1** [comment]
Ping.

File: azure/worker/tests/functions/return_http_404/function.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (pragnagopa) [inline azure/worker/tests/functions/return_http_404/function.json]: Nit: Move HttpTrigger tests to folder: HttpTrigger.  Will make it easier to navigate when you add more tests for other triggers

Comment (1st1) [reply azure/worker/tests/functions/return_http_404/function.json]: Good idea! I'll move them in a separate PR.

**Pull request merged**